googletest/test/gtest_unittest.cc

```diff
@@ -3319,38 +3321,45 @@ TEST_F(SingleEvaluationTest, ExceptionTests) {
   }, int);
   EXPECT_EQ(1, a_);
 
+  // successful EXPECT_THROW
+  EXPECT_THROW_WHAT({  // NOLINT
```

Comment:
How about no suffix at all? The _WHAT suffix was because I originally made the assumption that the exceptions would inherit from std::exception and implement what().